webapp/components/admin_console/configuration_settings.jsx

```diff
@@ -50,6 +51,7 @@ export default class ConfigurationSettings extends AdminSettings {
                     />
                 }
             >
+                <ReloadConfigButton/>
```

Comment:
Seems kind of weird to be dropping this into ConfigurationSettings, but I can't think of anywhere better to put it right now